src/components/common/ItemsNotFoundComponent.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Image } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import '../../_css/ItemsNotFoundComponent.css';
+
+
+const ItemsNotFoundComponent = ({ message, header }) => (
+  <div className="not-found">
+    <div className="middle">
+      <h2>{header}</h2>
+      <div className="center-image">
+        <Image
+          centered
+          src="images/empty.svg"
+          alt="empty box"
+          id="not_found_image"
+        />
+      </div>
+      <p id="not_found_message"> { message } </p>
```

Comment:
ALWAYS use **class** for styling, not id. Therefore, change the `id` to `class` and rename the class to `not-found__message`.

Also, delete the spaces before the opening and closing braces. Therefore, the p tag will be:
```
<p className="not-found__message">{message}</p>